packages/shared/src/components/fields/PortalMenu.tsx

```diff
@@ -3,19 +3,42 @@ import { Item, Menu, MenuProps } from '@dailydotdev/react-contexify';
 import { RootPortal } from '../tooltips/Portal';
 import ConditionalWrapper from '../ConditionalWrapper';
 import useContextMenu from '../../hooks/useContextMenu';
+import { useViewSize, ViewSize } from '../../hooks';
+import {
+  ContextMenuDrawer,
+  ContextMenuDrawerItem,
+} from '../drawers/ContextMenuDrawer';
 
 export default function PortalMenu({
   id,
   onHidden: onHiddenProps,
+  isOpen,
+  drawerOptions,
```

Comment:
Added this as a separate item from `options` because we have slight inconsistencies between the two interfaces, but I am open to consolidating these if everyone is happy with this approach

Reply:
I'm quite happy with this yes.
Basically if I remember correct the PortalMenu is only used as `ContextMenu` right? (Maybe we should also rename it at one stage)

But yes let's see what others think and we can clean it up a bit.